### PR TITLE
Update PA risk charts with PA specific risk categories

### LINKED ISSUE #491: Update PA risk chart with PA-specific categories
**What needs to be done? Why does it need to be done?**
PA has different risk level categories than MO. The risk level chart on the revocations matrix needs to be updated to reflect that. Charts that need to be updated
- Risk level chart
- Race chart
- Gender chart

The Risk Level column in the "Admitted individuals" list at the bottom should also reflect this state-specific mapping (show MEDIUM instead of MODERATE for PA).

**Additional context**
Add any other context about the task here.

### PR DESCRIPTION
**Pull request by jovergaag**
## Description of the change

PA and MO have different risk categories. This changes the risk categories for PA in the Risk Level, Gender, and Race charts, as well as in the Case Table.

I also added some constant files to use constants instead of string state codes and Core/Lantern state arrays.

## Type of change

- [ ] Bug fix (non-breaking change that fixes an issue)
- [x] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Configuration change (adjusts configuration to achieve some end related to functionality, development, performance, or security)

## Related issues

> Closes #491

### Development

These boxes should be checked by the submitter prior to merging:

- [x] Manual testing against realistic data has been performed locally

### Code review

These boxes should be checked by reviewers prior to merging:

- [x] This pull request has a descriptive title and information useful to a reviewer
- [x] This pull request has been moved out of a Draft state, has no "Work In Progress" label, and has assigned reviewers
- [x] Potential security implications or infrastructural changes have been considered, if relevant

### CODE REVIEW

File: src/tenants.js [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+// along with this program.  If not, see <https://www.gnu.org/licenses/>.
+// =============================================================================
+
+import * as lantern from "./utils/lanternConstants";
+import * as core from "./utils/coreConstants";
+
+export default {
+  [lantern.MO]: {
+    "name": "Missouri",
+    "avaiableStateCodes": [lantern.MO],
```
Comment (jessex) [inline src/tenants.js:24]: Whoooooops, typo: should be `availableStateCodes`

File: src/utils/transforms/labels.js [hunk 2]
```diff
@@ -18,32 +18,47 @@
...
-
-const riskLevelValuetoLabel = {
-  NOT_ASSESSED: 'Not Assessed',
-  LOW: 'Low',
-  MEDIUM: 'Moderate',
-  HIGH: 'High',
-  VERY_HIGH: 'Very high',
+]
+
+const riskLevelValueToLabelByStateCode = {
```
Comment (jessex) [inline src/utils/transforms/labels.js:31]: I think in the long-run we will want to declare state-specific manifests for these sorts of things, maybe even as an expansion of `tenants.js` or something like it, but I don't think it's a requirement just yet.

**Review by jessex — CHANGES_REQUESTED** [review]
Thanks, @jovergaag! I dig the cleanup/rationalization here. Just a nit and an idle thought from me.

File: src/utils/transforms/labels.js (hunk 2 above)
Comment (jovergaag) [reply src/utils/transforms/labels.js:31]: I totally agree. I was thinking of heading down that route in this PR but given the time pressure just wanted to make sure we can get this in before PA launch which *might?* be Thursday.

File: src/tenants.js (hunk 1 above)
Comment (jovergaag) [reply src/tenants.js:24]: great catch.

**Review by jessex — APPROVED** [review]
🙌

**Pull request merged**